content/en/tracing/trace_collection/dd_libraries/php.md

```diff
@@ -60,11 +60,20 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 {{< partial name="apm/apm-containers.html" >}}
 </br>
 
-3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
+3. After having instrumented your application, the tracing client sends traces to Unix domain socket `/var/run/datadog/apm.socket` by default. If the socket does not exist, traces are sent to `http://localhost:8126`.  If this is not the correct host and port change it by setting the below env variables:
 
-    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.
+    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` or `DD_TRACE_AGENT_URL`.
 
     See [environment variable configuration](#environment-variable-configuration) for more information on how to set these variables.
+
+Similarly, where `DD_TRACE_HEALTH_METRICS_ENABLED` is set true, the trace client attempts to send stats to the `/var/run/datadog/dsd.socket` Unix domain socket. If the socket does not exist then stats are sent to `http://localhost:8125`.
```

Comment:
`DD_TRACE_HEALTH_METRICS_ENABLED`. Interesting, I didn't know that feature. What is it about? 
It's not documented in the config page, should we document it there as well then?

Reply:
documented